assets/src/edit-story/components/helpCenter/menu/index.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+import { noop } from '../../../utils/noop';
+import { Footer } from './footer';
+import { Header } from './header';
+import { Tips } from './tips';
+import { Transitioner } from './transitioner';
+
+export function Menu({ onTipSelect = noop, ...transitionProps }) {
```

Comment:
Is the point of noop to define a default value? Seems a little wasteful, wouldn't an arrow function work here? 
```onTipSelect = () => {}, ```

Reply:
I'm honestly fine either way here. That's what I had earlier, but got a request to make it a util. I think either is fine tho unless anyone feels strongly about it. I certainly don't